plugin/scripts/highlight.js

```diff
@@ -63,17 +74,16 @@ function registerLongPauseHandler(e) {
 									e.target.value =
 										e.target.value +
 										res.Tracers[0].TracerPayload;
-									/* If the user uses the drop down for the first element, toggle the box on. */
-									toggleEnabled(e.target);
+									
+									simulateKeyPress(e.target);
```

Comment:
Can we also add a function that performs the onchange handler in addition to the keypress? Might look something like this:

```javascript
const evt = document.createEvent("HTMLEvents");
evt.initEvent("change", false, true);
element.dispatchEvent(evt);
```

I have a feeling that some jQuery will use the change handler instead of the keypress.